tests/test_dc2.py

```diff
@@ -87,17 +87,21 @@ def test_dc2_augmentation(self, cfg):
         aug_input_images = [imgs, bgs, deconv_image]
         aug_input_images = torch.cat(aug_input_images, dim=2)
 
-        aug_list = [
-            data_augmentation.aug_vflip,
-            data_augmentation.aug_rotate90,
-            data_augmentation.aug_rotate180,
-            data_augmentation.aug_rotate270,
-            data_augmentation.aug_shift,
-        ]
+        aug_list = [aug_vflip, aug_rotate90, aug_rotate180, aug_rotate270, aug_shift]
 
         for aug_method in aug_list:
             aug_image, aug_full = aug_method(origin_full, aug_input_images)
             assert aug_image[0, :, 0, :, :].shape == dc2_obj["images"].shape
             assert aug_image[0, :, 1, :, :].shape == dc2_obj["background"].shape
             assert aug_image[0, :, 2, :, :].shape == dc2_obj["deconvolution"].shape
             assert aug_full["n_sources"] <= origin_full.n_sources
+
+        # test rotatation
+        aug_image90, aug_full90 = aug_rotate90(origin_full, aug_input_images)
+        _, aug_full270 = aug_rotate270(origin_full, aug_input_images)
+
+        _, aug_full90180 = aug_rotate180(aug_full90, aug_image90)
+        _, aug_full90270 = aug_rotate270(aug_full90, aug_image90)
```

Comment:
This is a good test to have, but I realize it doesn't quite test what I thought it would, because there's no conversion btw tile and full catalogs happening.

Would you add an additional test that converts from a tile catalog to a full catalog and then back again to a tile catalog? (And that the first and last tile catalogs are equal?)